api/v1beta1/packetmachine_webhook.go

```diff
@@ -40,14 +41,14 @@ func (m *PacketMachine) SetupWebhookWithManager(mgr ctrl.Manager) error {
 // +kubebuilder:webhook:verbs=create;update,path=/mutate-infrastructure-cluster-x-k8s-io-v1beta1-packetmachine,mutating=true,failurePolicy=fail,matchPolicy=Equivalent,groups=infrastructure.cluster.x-k8s.io,resources=packetmachines,versions=v1beta1,name=default.packetmachine.infrastructure.cluster.x-k8s.io,sideEffects=None,admissionReviewVersions=v1;v1beta1
 
 // ValidateCreate implements webhook.Validator so a webhook will be registered for the type.
-func (m *PacketMachine) ValidateCreate() error {
-	machineLog.Info("validate create", "name", m.Name)
+func (m *PacketMachine) ValidateCreate() (admission.Warnings, error) {
+	machineLog.Info("PacketMachine.ValidateCreate called (not implemented)", "name", m.Name)
 
-	return nil
+	return nil, nil
 }
 
 // ValidateUpdate implements webhook.Validator so a webhook will be registered for the type.
-func (m *PacketMachine) ValidateUpdate(old runtime.Object) error {
+func (m *PacketMachine) ValidateUpdate(old runtime.Object) (admission.Warnings, error) {
 	machineLog.Info("validate update", "name", m.Name)
```

Comment:
nit: for consistency with the other info changes, we could say `PacketMachine.ValidateUpdate called`

Reply:
presumably, there are more occurrences